### PR TITLE
Remove openvino-extensions package

### PR DESCRIPTION
**Pull request by dkurt**
`openvino-extensions` is a custom package designed for https://github.com/openvinotoolkit/openvino_notebooks/pull/46/. As the demo reverted, do not install it.

### CODE REVIEW

**Comment by ryanloney** [comment]
Thanks @dkurt. We hope to host the facies notebook when the model weights have a reliable storage location, like Amazon S3. For now we can remove openvino-extensions dependency though.

**Comment by dkurt** [comment]
Hi! I have issues with using `pipenv` (it hangs without response). So if you are able, feel free to reopen a PR.

**Comment by helena-intel** [comment]
I locked the Pipfile, but it seems there is an unrelated upstream issue with the Docker image now. I'll keep an eye on it. Once the tests pass I'll merge the PR (and if there is still an issue I'll fix it).

I'm sad that we cannot have the facies notebook at the moment. I still hope we can add it soon.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -155,15 +156,30 @@ To maintain consistency between notebooks, please follow the directory structure
...
 In case of output provided by Notebook please create folder ```output``` on the same level as readme file.
 
+### Notebook utils
+
+The _notebook_utils.py_ file in the _notebooks/utils_ directory contains utility functions and classes that can be reused across
+notebooks. It contains a `download_file()` function that optionally shows a progress bar, and a standard way to convert
+segmentation maps to images and display them. The Python file is generated from _notebook_utils.ipynb_ notebook in the same directory.
+If you want to add a function or class to _notebook_utils.py_, please add it to the notebook, and generate the
+Python file with `jupyter nbconvert notebook_utils.ipynb --TagRemovePreprocessor.remove_cell_tags=hide --to script`
+Add a "hide" tag to any demo cells (from the right side gear sidebar) to prevent these cells from being added to the script.
```
Comment (dkurt) [inline CONTRIBUTING.md:166]: @helena-intel, Seems like there are unrelated changes

Comment (helena-intel) [reply CONTRIBUTING.md:166]: @dkurt I rebased to main, the changes should all be from that. I'll doublecheck before merging.
